src/store/DevfileRegistries/index.ts

```diff
@@ -173,7 +173,7 @@ export const actionCreators: ActionCreators = {
 
         const schemav200 = await WorkspaceClient.restApiClient.getDevfileSchema('2.0.0');
         const schemav210 = await WorkspaceClient.restApiClient.getDevfileSchema('2.1.0');
-        const schemav220alpha = await WorkspaceClient.restApiClient.getDevfileSchema('2.2.0-alpha');
+        const schemav220alpha = await WorkspaceClient.restApiClient.getDevfileSchema('2.2.0');
```

Comment:
maybe rename the constant :-)

Reply:
I can do it but then I need a comment that would explain that it's not released yet as I have in Che Server https://github.com/eclipse-che/che-server/pull/27/files#diff-b1166dc0647d4d5da93d2d6753d2281d94182ab9f5a4a91bc58854d1436abdefR31 )